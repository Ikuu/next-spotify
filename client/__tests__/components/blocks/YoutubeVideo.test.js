import React from 'react';
import { create } from 'react-test-renderer';
import YoutubeVideo from '../../../components/blocks/YoutubeVideo';

test('YoutubeVideo renders', () => {
  const tree = create(<YoutubeVideo id="tMgkt9jdjTU" title="Test" />);
  expect(tree).toMatchSnapshot();
});
