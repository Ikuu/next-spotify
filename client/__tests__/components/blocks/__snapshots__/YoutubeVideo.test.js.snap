// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YoutubeVideo renders 1`] = `
<div
  style={
    Object {
      "height": 0,
      "maxWidth": "100%",
      "overflow": "hidden",
      "paddingBottom": "56.25%",
      "position": "relative",
    }
  }
>
  <iframe
    allow="autoplay; encrypted-media"
    allowFullScreen={true}
    frameBorder="0"
    src="https://www.youtube.com/embed/tMgkt9jdjTU"
    style={
      Object {
        "bottom": 0,
        "height": "100%",
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
        "width": "100%",
      }
    }
    title="Test"
  />
</div>
`;
